modules/machine-vsphere-machines.adoc

```diff
@@ -39,14 +39,16 @@ Ensure that all virtual machine names across a vSphere installation are unique.
 ====
 .. On the *Select a name and folder* tab, select the name of the folder that you created for the cluster.
 .. On the *Select a compute resource* tab, select the name of a host in your datacenter.
+.. On the *Select storage* tab, select storage for your configuration and disk files.
 .. On the *Select clone options*, select *Customize this virtual machine's hardware*.
-.. On the *Customize hardware* tab, click *VM Options* -> *Advanced*.
-*** Click *Edit Configuration*, and on the *Configuration Parameters* window, click *Add Configuration Params*. Define the following parameter names and values:
+.. On the *Customize hardware* tab, click *Advanced Parameters*.
+*** Add the following configuration parameter names and values by specifying data in the *Attribute* and *Values* fields. Ensure that you select the *Add* button for each parameter that you create.
 **** `guestinfo.ignition.config.data`: Paste the contents of the base64-encoded compute Ignition config file for this machine type.
 **** `guestinfo.ignition.config.data.encoding`: Specify `base64`.
 **** `disk.EnableUUID`: Specify `TRUE`.
```

Comment:
Wondering what you think about this as an alternative to reduce the nesting a bit:
```suggestion
.. Add the following configuration parameter names and values by specifying data in the *Attribute* and *Values* fields. Ensure that you select the *Add* button for each parameter that you create.
*** `guestinfo.ignition.config.data`: Paste the contents of the base64-encoded compute Ignition config file for this machine type.
*** `guestinfo.ignition.config.data.encoding`: Specify `base64`.
*** `disk.EnableUUID`: Specify `TRUE`.
```